spec/unit/mixin/powershell_out_spec.rb

```diff
@@ -44,27 +44,21 @@
       ).and_return(ret)
       expect(object.powershell_out("Get-Process", timeout: 600)).to eql(ret)
     end
-  end
 
-  describe "#powershell_out!" do
-    it "runs a command and returns the shell_out object" do
-      mixlib_shellout = double("Mixlib::ShellOut")
-      expect(object).to receive(:shell_out).with(
-        "powershell.exe #{flags} -Command \"Get-Process\"",
-        {}
-      ).and_return(mixlib_shellout)
-      expect(mixlib_shellout).to receive(:error!)
-      expect(object.powershell_out!("Get-Process")).to eql(mixlib_shellout)
-    end
+    context "when double quote is passed in the powershell command" do
+      it "passes if double quote is appended with single escape", :windows_only do
+        ret = double("Mixlib::ShellOut")
+        expect(object).to receive(:shell_out).with(
+          "powershell.exe #{flags} -Command \"Write-Verbose \\\"Some String\\\" -Verbose\"",
+          {}
+        ).and_return(ret)
+        expect(object.powershell_out("Write-Verbose \"Some String\" -Verbose")).to eql(ret)
+      end
 
-    it "passes options" do
-      mixlib_shellout = double("Mixlib::ShellOut")
-      expect(object).to receive(:shell_out).with(
-        "powershell.exe #{flags} -Command \"Get-Process\"",
-        timeout: 600
-      ).and_return(mixlib_shellout)
-      expect(mixlib_shellout).to receive(:error!)
-      expect(object.powershell_out!("Get-Process", timeout: 600)).to eql(mixlib_shellout)
+      it "raises error if double quote is passed with double escape characters", :windows_only do
+        res = object.powershell_out("Write-Verbose \\\"Some String\\\" -Verbose")
+        expect(res.stderr).to include("A positional parameter cannot be found that accepts argument")
```

Comment:
Can we catch this error and raise one that says you're probably using double escaped double quotes?